src/Documentation.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace PhpSchool\Website;
+
+use ArrayIterator;
+use IteratorAggregate;
+
+/**
+ * Class Documentation
+ * @package PhpSchool\Website
+ * @author Aydin Hassan <aydin@hotmail.co.uk>
+ */
+class Documentation implements IteratorAggregate
+{
+    /**
+     * @var DocumentationGroup[]
+     */
+    private $groups = [];
+
+    /**
+     * @param DocumentationGroup $group
+     */
+    public function addGroup(DocumentationGroup $group)
+    {
+        $this->groups[] = $group;
+    }
+
```

Comment:
that's what I was thinking, 7 is self documenting (when you use it like that anyway)

only thing I'd say is  consistency... eg method above has it and others don't .. happy to go with either tbh